### PR TITLE
Added check for temperatures

### PR DESCRIPTION
**Pull request by jordy25519**
## Pull Request Checklist

**Is this in reference to an existing issue?**
No
#### General
- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
- [x] Update README with any necessary configuration snippets
- [] Binstubs are created if needed
- [x] RuboCop passes
- [x] Existing tests pass 
#### New Plugins
- [ ] Tests
- [x] Add the plugin to the README
- [x ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)
#### Purpose

Provide temperature checking capability

### CODE REVIEW

**Comment by eheydrick** [comment]
Looks good. Can you rebase so we can merge.

**Comment by GhostLyrics** [comment]
@holygits Please be so kind as to also check the new readme format. If there are any questions I'll be happy to guide you through this.

**Comment by jordy25519** [comment]
I've made my best attempt at conforming to the new readme format.

File: bin/check-temperature.rb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  def run
+    raw = `sensors`
+    sections = raw.split("\n\n")
+    metrics = {}
+    sections.each do |section|
+      section.split("\n").drop(1).each do |line|
+        begin
+          key, value = line.split(':')
+          key = key.downcase.gsub(/\s/, '')
+          if key[0..3] == 'temp' || key[0..3] == 'core'
```
Comment (GhostLyrics) [inline bin/check-temperature.rb]: I suggest also including `loc` as a key here.

Example from one of my machines:
```plain
root@machine:~# sensors
i350bb-pci-0500
Adapter: PCI adapter
loc1:         +63.0°C  (high = +120.0°C, crit = +110.0°C)

coretemp-isa-0000
Adapter: ISA adapter
Physical id 0:  +32.0°C  (high = +86.0°C, crit = +100.0°C)
Core 0:         +28.0°C  (high = +86.0°C, crit = +100.0°C)
<REST OMITTED>
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,13 +3,15 @@ This project adheres to [Semantic Versioning](http://semver.org/).
...
 
+
 ## [Unreleased]
 
 ## [1.0.0] - 2017-03-15
 ### Added
 - made metrics aware of multiple chips (@GhostLyrics)
 - made metrics also collect power draw (@GhostLyrics)
 - support for Ruby 2.3.0 (@eheydrick)
+- added temperature check (@holygits)
```
Comment (GhostLyrics) [inline CHANGELOG.md]: Your change should be in Unreleased since 1.0.0 was already released without your PR a few days ago.

**Review by GhostLyrics — APPROVED** [review]
I would like to thank you for the new check, @holygits.

The requested changes are mostly cosmetic. We're almost good to go.

File: bin/check-temperature.rb (hunk 1 above)
Comment (GhostLyrics) [reply bin/check-temperature.rb]: You could also include `physical` but I don't know enough about the workings of `sensors` to know for sure.

**Comment by GhostLyrics** [comment]
Thank you for your contribution.

**Comment by eheydrick** [comment]
The new check is in 1.1.0 now live on Rubygems.

**Pull request merged**